ads/google/csa.md

```diff
@@ -15,6 +15,9 @@ limitations under the License.
 -->
 # Custom Search Ads
 
+_The Custom Search Ads integration is offered as a Beta and is subject to the
+conditions of the Google Program Guidelines._
```

Comment:
AFS is offered as an Direct and Online product with two sets of GPGs. Can we leave the statement w/o a link so users can interpret it according to their own GPGs? If that's not possible, let's exclude the part referring to the GPGs and just say "The Custom Search Ads integration is offered as a Beta."